src/popup/components/buttons/BtnMain.vue

```diff
@@ -80,6 +80,7 @@ export default defineComponent({
   @extend %face-sans-16-regular;
 
   position: relative;
+  z-index: 1;
```

Comment:
Why `z-index` was introduced?

Reply:
Because the `before` pseudo element has the z-index `-1` so if the parent has no positive z-index the `before` would not be visible (will be placed "under" the body.